### PR TITLE
QA fixes for merged PRs (189, 188, 187)

### PR DESCRIPTION
**Pull request by dbojicic**
QA fixes for merged PRs into GitHub. 

Fixes https://github.com/hl7au/au-fhir-core/issues/189
Fixes https://github.com/hl7au/au-fhir-core/issues/188
Fixes https://github.com/hl7au/au-fhir-core/issues/187

+ additional changes to change log to remove entries for non-substantive changes.

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-condition-intro.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 - `Condition.code` contains the identification of the condition, problem or diagnosis which may include body site information. Where a system has information not supported by the coding in `Condition.code.coding` (e.g. body site, laterality and other qualification of condition coding terms) that information **SHOULD** be supplied in `Condition.code.text`
 - An active condition is represented using "active" in `Condition.clinicalStatus`
 - To represent that a patient does not have a condition or history of condition, an appropriate negation code is used in `Condition.code`:
-  - For indicating no known conditions or problems of any type `Condition.code` **SHOULD** use the code `SNOMED CT: 160245001 | No current problems or disability (situation)`
+  - For indicating no known conditions or problems of any type `Condition.code` **SHOULD** use the SNOMED-CT 160245001 \|No current problems or disability (situation)\|
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-condition-intro.md:28]: It is "SNOMED CT"  not "SNOMED-CT"

File: input/intro-notes/StructureDefinition-au-core-heartrate-intro.md [hunk 2]
```diff
@@ -19,4 +19,4 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 
 ### Profile specific implementation guidance
 {% include observation_vitalsigns_guidance.md -%}
-- Pulse rate and heart rate are different concepts. In FHIR, both concepts are children of the parent Heart Rate concept and conform to the AU Core Heart Rate profile. When representing a pulse rate observation specifically, an *additional* SNOMED-CT 78564009 Pulse rate code **SHALL** be added to `Observation.code`. This allows pulse rate observations to be identified and searched for directly. When populating pulse rate, the body site from which the pulse rate was observed is optional to include.
+- Pulse rate and heart rate are different concepts. In FHIR, both concepts are children of the parent Heart Rate concept and conform to the AU Core Heart Rate profile. When representing a pulse rate observation specifically, an *additional* SNOMED-CT code 78564009 \|Pulse rate\| **SHALL** be added to `Observation.code`. This allows pulse rate observations to be identified and searched for directly. When populating pulse rate, the body site from which the pulse rate was observed is optional to include.
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-heartrate-intro.md:22]: "SNOMED CT" not "SNOMED-CT"

File: input/pagecontent/changes.md [hunk 3]
```diff
@@ -76,11 +76,9 @@ This change log documents the significant updates and resolutions implemented fr
 - Updated mapping of AUCDI Procedure Completed Event to be represented by Procedure only [au-fhir-core #147](https://github.com/hl7au/au-fhir-core/issues/147).
 - Updated mapping of AUCDI Medication Use Statement to be represented by MedicationStatement only [au-fhir-core #145](https://github.com/hl7au/au-fhir-core/issues/145).
 - Made the following changes to AU Core Condition:
-  - Removed Must Support from the following elements in AU Core Condition:
-    - Condition.asserter [FHIR-45019](https://jira.hl7.org/browse/FHIR-45019), [FHIR-45127](https://jira.hl7.org/browse/FHIR-45127), [FHIR-45083](https://jira.hl7.org/browse/FHIR-45083)
-    - Condition.recorder [FHIR-45019](https://jira.hl7.org/browse/FHIR-45019), [FHIR-45082](https://jira.hl7.org/browse/FHIR-45082)
-    - Condition.encounter [FHIR-45189](https://jira.hl7.org/browse/FHIR-45189)
-  - Condition [FHIR-45058](https://jira.hl7.org/browse/FHIR-45058) additional narrative with guidance on *no known problem* coding.
```
Comment (dt-r) [inline input/pagecontent/changes.md:83]: restore change log line for no known problem coding but make better so it is "Added profile-specific implementation guidance on representing no known problem [FHIR-45058](https://jira.hl7.org/browse/FHIR-45058)"

File: input/pagecontent/changes.md [hunk 4]
```diff
@@ -113,6 +111,7 @@ This change log documents the significant updates and resolutions implemented fr
   - removed Must Support from Observation.encounter [FHIR-45134](https://jira.hl7.org/browse/FHIR-45134)
   - removed Must Support from Observation.performer [FHIR-44786](https://jira.hl7.org/browse/FHIR-44786) 
   - removed the fixed value constraint 'final' on Observation.status [FHIR-45120](https://jira.hl7.org/browse/FHIR-45120)
+  - added profile-specific implementation guidance on representing a specific pulse rate in the heart rate profile [FHIR-44861](https://jira.hl7.org/browse/FHIR-44861)
```
Comment (dt-r) [inline input/pagecontent/changes.md:114]: Change "added profile-specific implementation guidance on representing a specific pulse rate in the heart rate profile" to be "added profile-specific implementation guidance on representing pulse rate observations"

**Review by dt-r — CHANGES_REQUESTED** [review]
See comments for requested changes.

File: input/intro-notes/StructureDefinition-au-core-heartrate-intro.md (hunk 2 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-heartrate-intro.md]: Fixed.

File: input/intro-notes/StructureDefinition-au-core-condition-intro.md (hunk 1 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-condition-intro.md]: Fixed.

File: input/pagecontent/changes.md (hunk 4 above)
Comment (dbojicic) [reply input/pagecontent/changes.md]: Fixed.

File: input/pagecontent/changes.md (hunk 3 above)
Comment (dbojicic) [reply input/pagecontent/changes.md:83]: Fixed.

File: input/intro-notes/StructureDefinition-au-core-condition-intro.md [hunk 5]
```diff
@@ -25,7 +25,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 - `Condition.code` contains the identification of the condition, problem or diagnosis which may include body site information. Where a system has information not supported by the coding in `Condition.code.coding` (e.g. body site, laterality and other qualification of condition coding terms) that information **SHOULD** be supplied in `Condition.code.text`
 - An active condition is represented using "active" in `Condition.clinicalStatus`
 - To represent that a patient does not have a condition or history of condition, an appropriate negation code is used in `Condition.code`:
-  - For indicating no known conditions or problems of any type `Condition.code` **SHOULD** use the code `SNOMED CT: 160245001 | No current problems or disability (situation)`
+  - For indicating no known conditions or problems of any type `Condition.code` **SHOULD** use the SNOMED CT 160245001 \|No current problems or disability (situation)\|
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-condition-intro.md:28]: grammar issue remove "the" so it is "use SNOMED CT"

File: input/intro-notes/StructureDefinition-au-core-condition-intro.md (hunk 1 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-core-condition-intro.md]: noted fix. there is a grammar issue. restore the word "code" so it becomes "the code SNOMED CT"

File: input/pagecontent/changes.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/changes.md:83]: Fix the copy paste spelling error. "problems" instead of "problem:

**Review by dt-r — CHANGES_REQUESTED** [review]
more tweaks

File: input/intro-notes/StructureDefinition-au-core-condition-intro.md (hunk 1 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-condition-intro.md]: Done.

**Review by dt-r — APPROVED** [review]

**Pull request merged**